app/models/casa_case.rb

```diff
@@ -36,7 +38,7 @@ class CasaCase < ApplicationRecord
   belongs_to :judge, optional: true
   belongs_to :casa_org
   validates :birth_month_year_youth, presence: true
-
+  validates_presence_of :casa_case_contact_types, message: ": At least one contact type must be selected", if: :validate_contact_type
```

Comment:
interesting phrasing here `": At` for error composition I assume

Reply:
Yeah its for error composition to have something like '_attribute_name : error_message_'